src/components/PageBackendError/index.js

```diff
@@ -25,7 +28,7 @@ const PageBackendError = ({error}) => (
           <p className='backend-error__message'>{error.config.data}</p>
         </> : null}
     </div>
-  </>
+  </Trans>
```

Comment:
This is a section where I am not sure if replacing the empty div with the Trans macro is better or not. This is also a section where I'm not sure if the whole translation should be one big item or multiple smaller ones.

Reply:
I agree with what you've done here. If someone sees this page we really want them to screengrab it and send it to us. I don't think the empty divs serve any purpose other than keeping React happy because of the Helmet component.